sites/public/layouts/eligibility.tsx

```diff
@@ -2,14 +2,54 @@ import Layout from "./application"
 import styles from "./Eligibility.module.scss"
 import { ApplicationTimeout } from "../src/forms/applications/ApplicationTimeout"
 import React, { useContext } from "react"
-import { ProgressNav, t } from "@bloom-housing/ui-components"
+import {
+  AppearanceStyleType,
+  Button,
+  Form,
+  FormCard,
+  ProgressNav,
+  t,
+} from "@bloom-housing/ui-components"
 import { EligibilityContext } from "../lib/EligibilityContext"
 import { ELIGIBILITY_SECTIONS } from "../lib/constants"
 import { eligibilityRoute } from "../lib/helpers"
+import { UseFormMethods } from "react-hook-form"
+import { useRouter } from "next/router"
+import { getFilterUrlLink } from "../lib/filterUrlLink"
 
-const EligibilityLayout = (props) => {
+export interface EligibilityLayoutProps {
+  title: string
+  currentPage: number
+  children: any
+  onFinishStep?: (data: any) => void
+  formMethods: UseFormMethods
+}
+
+const EligibilityLayout = (props: EligibilityLayoutProps) => {
+  const router = useRouter()
   const { eligibilityRequirements } = useContext(EligibilityContext)
 
+  const handleSubmit = props.formMethods.handleSubmit
+
+  const onBack = async (data) => {
+    props.onFinishStep && props.onFinishStep(data)
```

Comment:
minor nit: I think this format is convenient when you're trying to add html elements but since this is pure typescript I think it's clearer if you expand this:
```
if (props.onFinishStep) {
  props.onFinishStep(data)
}
```